Assets/Scripts/BodyPart.cs

```diff
@@ -13,21 +14,21 @@ public class BodyPart : MonoBehaviour
 {
     public BotController botController;
     public int maxHitpoints;
-    private int hitpoints;
+    protected int hitpoints;
     public BodyPartState state;
 
     // Start is called before the first frame update
     public void Start()
     {
-        botController = GetComponentInParent<BotController>();
+        maxHitpoints = 100;
```

Comment:
We should probably let this be controlled by the inspector ultimately (but hard-setting to 100 for testing is fine)

Reply:
oh this will overwrite a value set in the editor right? If i want a code default I should do it directly on the field rather than in start?